src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs

```diff
@@ -22,7 +23,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return s_frameworkDescription ?? (s_frameworkDescription = $"{FrameworkName} {typeof(object).GetTypeInfo().Assembly.GetName().Version}");
+                return s_frameworkDescription ?? (s_frameworkDescription = $"{FrameworkName} {FileVersionInfo.GetVersionInfo(typeof(object).GetTypeInfo().Assembly.Location).FileVersion}");
```

Comment:
You could avoid the new reflection dependency (and using NETStandard1.5 which causes this problem) by instead just using reflection to get the assembly level attribute.  Any use of reflection here (the current one, as well as the one I am proposing) should be validated in NETNative to make sure the metadata is preserved.  If not we'll need to add an rd.xml.